packages/x-charts/src/ScatterChart/Scatter.types.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { ScatterMarkerOwnerState, ScatterMarkerProps } from './ScatterMarker';
+import { useInteractionItemProps } from '../hooks/useInteractionItemProps';
+
+export interface ScatterMarkerSlots {
+  /**
+   * The component that renders the marker for a scatter point.
+   * @default ScatterMarker
+   */
+  marker?: React.JSXElementConstructor<ScatterMarkerProps>;
+}
+
+export interface ScatterMarkerSlotProps {
+  marker?: ScatterMarkerProps;
+}
+
+export interface ScatterMarkerElementProps extends ScatterMarkerOwnerState {
```

Comment:
the `ScatterMarkerElementProps` probably makes sense to be on the same place as `ScatterMarkerElement` is. Would probably move all these files into `ScatterMarker.ts/.types.ts`